samples/nrf9160/http_update/common/src/update.c

```diff
@@ -200,6 +229,19 @@ int update_sample_init(struct update_sample_init_params *params)
 	return 0;
 }
 
+void update_sample_start(void)
+{
+	int err;
```

Comment:
Should we have a check here to verify that `filename` is not null? Even though this is a sample-only subsys I guess it could be useful.

Reply:
Yes, good idea